ktlint-ruleset-standard/src/test/resources/spec/method-expression-body/format.kt.spec

```diff
@@ -0,0 +1,15 @@
+class C0 {
+    fun good(): Int = funcReturningInt()
+    fun bad(): Int {
+        return funcReturningInt()
```

Comment:
Thank you for building auto correcting of this rule!
Please take a look at some additional test case we have starting https://github.com/arturbosch/detekt/blob/04a9ef332de2ee5bd45c4be9bd480bf28ecabd01/detekt-rules/src/test/kotlin/io/gitlab/arturbosch/detekt/rules/style/ExpressionBodySyntaxSpec.kt#L58-L66.
Be aware that some of our users (and maybe also KtLint users) prefer not to use the expression syntax for functions if the return statement reaches multiple lines:
```
fun stuff(arg: Int): Int {
    return when(arg) {
        0 -> 0
		else -> 1
    }
}
```

Reply:
Applying the rule in that case seems like what you'd want if this rule was turned on, no?
```
fun stuff(arg: Int): Int = when(arg) {
        0 -> 0
	else -> 1
    }
}
```

Reply:
Is there a mechanism in ktlint to apply a sort of 'sub option' to a rule, or if we wanted to allow the user to apply this but not for multi-line (like above), would it have to be a separate rule?

Reply:
@bbaldino there is not. The Android guide says:
```
Otherwise, if an expression function grows to require wrapping, use a normal function body, a return declaration, and normal expression wrapping rules instead.
```
So perhaps you can make this even stricter that it will only fail the rule if the expression is also a single *line*.

Reply:
ok, i'll take a look at that.